crawl-ref/source/decks.cc

```diff
@@ -1326,43 +1326,14 @@ static void _summon_dancing_weapon(int power)
     mon->ghost_demon_init();
 }
 
-static void _summon_flying(int power)
+static void _summon_bee(int power)
 {
     const int power_level = _get_power_level(power);
-
-    const monster_type flytypes[] =
-    {
-        MONS_WYVERN, MONS_KILLER_BEE,
-        MONS_VAMPIRE_MOSQUITO, MONS_HORNET
-    };
-    const int num_flytypes = ARRAYSZ(flytypes);
-
-    // Choose what kind of monster.
-    monster_type result;
-    const int how_many = 2 + random2(3) + power_level * 3;
-    bool hostile_invis = false;
-
-    do
-    {
-        result = flytypes[random2(num_flytypes - 2) + power_level];
-    }
-    while (is_good_god(you.religion) && result == MONS_VAMPIRE_MOSQUITO);
+    const monster_type flytypes = MONS_KILLER_BEE;
+    const int how_many = 2 + random2(3) + ((power_level * 2) + 1) * 3;
```

Comment:
```suggestion
    const int how_many = 1 + random2((power_level + 1) * 3);
```

It's the meat shield card but it's not the reliable meat shield card. Reliable shielding is more an escape thing. 15 bees as a max roll is also too many.